Source/WebKit/NetworkProcess/storage/CacheStorageCache.cpp

```diff
@@ -38,9 +38,12 @@ namespace WebKit {
 
 static String computeKeyURL(const URL& url)
 {
+    RELEASE_ASSERT(url.isValid() && !url.isEmpty());
```

Comment:
It is best to have separate assertions so we know which check is failing in case of a crash.